test/parsing_test.js

```diff
@@ -533,10 +529,10 @@ vows
         };
       },
       "large one doesn't parse": function(c) {
-        assert.equal(c.cookie1, null);
+        assert.equal(c.cookie1, undefined);
```

Comment:
To be absolutely certain the value is `undefined`.
```suggestion
        assert.strictEqual(c.cookie1, undefined);
```